Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2778,17 +2778,6 @@ def gen2018HiMix(fragment,howMuch):
     '--era' : 'Run2_2016',
     '--filetype':'DQM',
     }
-steps['NANOUP15']={
-    '--mc':'',
-    '--step':'NANO', # todo: remove UP from label
-    '--nThreads':'2',
```

Comment:
@ahmad3213 @chayanit the setting for the number of threads is lost in the redefinition, is this done on purpose and why, if this is the case?

Reply:
nthreads = 2 was copied from production but for relvals 1 should be enough. NanoAOD should run very fast.  @fabiocos Do you have suggestion?

Reply:
@ahmad3213 I am not concerned about speed, I think there is a value in testing thread safety by running multi-threaded, you might get problems that you don't in single thread mode

Reply:
i have added back '--nThreads': '2'